src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -771,6 +783,19 @@ def ac_user_delete_cmd(_, username):
         return -errno.ENOENT, '', str(ex)
 
 
+@CLIWriteCommand('dashboard ac-user-attempt-reset',
+                 'name=username,type=CephString',
+                 'Disable a user')
+def ac_user_attempt_reset(_, username):
+    try:
+        mgr.ACCESS_CTRL_DB.get_user(username)
+        mgr.ACCESS_CTRL_DB.save_attempt(username, 0)
+        mgr.ACCESS_CTRL_DB.save()
+        return 0, "Attempts resetted for the user '{}'".format(username), ''
+    except UserDoesNotExist as ex:
+        return -errno.ENOENT, '', str(ex)
```

Comment:
This wouldn't be needed if we rely only on the enabled property. That ensures the user management workflow can be carried out either from CLI (`ac-user-disable`) or UI (admin -> User Management).

Reply:
Removed the CLI command. Instead when the user is enabled the attempt counter will get resets to 0.

Reply:
That's ok too!